libraries/src/Console/UpdateCoreCommand.php

```diff
@@ -386,4 +400,34 @@ public function copyFileTo($file, $dir): void
     {
         Folder::copy($file, $dir, '', true);
     }
+
+    /**
+     * Check the schema version
```

Comment:
```suggestion
     * Check Database Table Structure
```
Comment should be adapted.